pkg/storage/segments/store.go

```diff
@@ -113,6 +113,7 @@ func (s *segmentStore) Put(ctx context.Context, data io.Reader, expiration time.
 		}
 	} else {
 		// uses overlay client to request a list of nodes
+		// TODO(moby) update Space
```

Comment:
what is update Space?

Reply:
It's the param passed into the overlay below. I'll update the comment to be clearer